### DISCUSSION TITLE
Load Images from assets folder

### DISCUSSION THREAD

**Discussion by FADL285** [nuxt/image#934] [Q&A]
When Loading Images from `assets ` the image doesn't load

```vue
<script setup lang="ts">
import NotFoundImage from "~/assets/images/404.png"
</script>

<template>
  <NuxtImg
        :src="NotFoundImage"
        alt="404 error, Page Not Found"
   />
</template>    
```

**Comment by notflip** [comment]
I'm also wondering what the default folder for the IPX provider is, how can something like this be missing from the docs?

**Reply by Sadarex** [reply]
> I'm also wondering what the default folder for the IPX provider is, how can something like this be missing from the docs?

The default value is `public`
https://image.nuxtjs.org/configuration#dir

**Comment by Sadarex** [comment]
Did you change the value for **dir**?

https://image.nuxtjs.org/configuration#dir
```
export default defineNuxtConfig({
  image: {
    dir: 'assets/images'
  }
})
```

**Comment by cayque10** [comment]
I have se same problem. In develop funciona, but in production the directory is the public path.

`
 image: {
    provider: 'ipx',
    dir: 'assets/image',
    presets: {
      default: {
        modifiers: {
          format: 'webp',
          loading: 'lazy',
          quality: "100"
        }
      }
    },
  }
`
**Production**
IPX Error: Error: File not found (/root/jpd/site/.output/public/16919377384219775810.png)

**development (I forced the error to see a folder, but it works)**
IPX Error: Error: File not found (C:/Users/Jaca/jpd/assets/img/1691939023680253d6209.png)


How is it possible to still be picking up from the public folder?

**Reply by cayque10** [reply]
Analyzing the source code, I noticed that in production the public path is always used. If it's possible to set the path to IPX, what's the point of calling only the public path in production mode?

![ipx](https://github.com/nuxt/image/assets/17622660/1c06fd38-1c51-408d-9588-5140b14b2192)


I made a change for testing purposes. And by the way, would that make sense?

**Reply by cayque10** [reply]
It seems that the fix has already been released, but apparently it is not available to update through NPM.

https://github.com/nuxt/image/commit/7e187e5

https://github.com/nuxt/image/releases

**Comment by Sadarex** [comment]
It should work with the latest edge release. Add `"@nuxt/image-edge": "latest"` in your package.json
